features/support/env.local.rb

```diff
@@ -106,6 +106,9 @@ def tournament_named(name)
   end
 
   def single_tournament
+    if @tournaments.empty?
```

Comment:
Um, no. You should use a different method for form created objects.

Reply:
why?